spec/shared_behaviors/taggable.rb

```diff
@@ -139,7 +149,12 @@ def tag_link(s)
         '#-initialhyphen'        => '-initialhyphen',
         '#-initialhyphen tag'    => '-initialhyphen',
         '#-initial-hyphen'       => '-initial-hyphen',
-        "\u202a#\u200eUSA\u202c" => 'USA'
+        "\u202a#\u200eUSA\u202c" => 'USA',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.